packages/charts/test/chart-properties.test.js

```diff
@@ -568,4 +568,42 @@ describe('vaadin-chart properties', () => {
       });
     });
   });
+
+  describe('annotations', () => {
+    let chart;
+
+    beforeEach(async () => {
+      chart = fixtureSync('<vaadin-chart></vaadin-chart>');
+      await oneEvent(chart, 'chart-load');
+    });
+
+    it('should not have annotations by default', () => {
+      expect(chart.configuration.options.annotations).to.be.empty;
+    });
+
+    it('should have annotations when annotation label is set using update', async () => {
+      chart.updateConfiguration(
+        {
+          annotations: [
+            {
+              labels: [
+                {
+                  text: 'Sample text',
+                  point: {
+                    x: 50,
+                    y: 50,
+                  },
+                },
+              ],
+            },
+          ],
+        },
+        true,
+      );
+      await aTimeout(50);
+      expect(chart.configuration.options.annotations.length).to.be.equal(1);
+      expect(chart.configuration.options.annotations[0].labels.length).to.be.equal(1);
+      expect(chart.configuration.options.annotations[0].labels[0].text).to.be.equal('Sample text');
```

Comment:
Also, check that the annotation is actually there. Otherwise the test will pass even without the `import 'highcharts/es-modules/masters/modules/annotations.src.js';`

```suggestion
      expect(chart.configuration.options.annotations[0].labels[0].text).to.be.equal('Sample text');
      
      const annotationText = chart.$.chart.querySelector('.highcharts-annotation');
      expect(annotationText.textContent).to.be.equal('Sample text');
```